### ISSUE TITLE
Cannot zoom in on side panel

### ISSUE THREAD

**Issue by KorigamiK** [zen-browser/desktop#1516]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Ctrl+/- zooms the active tab. There's no way to zoom in on the side panel

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1

### What platform are you seeing the problem on?

Linux

**Comment by guigirl42** [comment]
Yes, i miss the ability to **zoom in/out** there too. `Floorp` can do it, so obviously there is a way that's compatible with Firefox [i mean, Vivaldi also can do it, but that's only a chromium base].

**Comment by guigirl42** [comment]
Given that _weird_ bot post [& associated email] above, i'm posting this to keep the issue alive & restate that **this function would be an excellent enhancement**. Pls _bring Zen into parity with Floorp & Vivaldi_ in this regard.

**Comment by salamijason** [comment]
Hey! Good suggestion. Rather than a bug, this leans more on the something that the devs just didn't think about so I'll move this over to discussions. Thank you!
